image-viewer.py

```diff
@@ -56,72 +59,87 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
-        
+
     def create_menu(self):
         menubar = Menu(self.root)
         self.root.config(menu=menubar)
-        
+
         file_menu = Menu(menubar, tearoff=False)
-        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0)
-        file_menu.add_command(label='Refresh', command=self.refresh, underline=0)
-        file_menu.add_command(label='Exit', command=self.root.destroy, underline=1)
-        menubar.add_cascade(label='File', menu=file_menu, underline=0)
-    
+        file_menu.add_command(label="Open file(s)", command=self.open_files)
+        file_menu.add_command(label="Open Folder", command=self.open_folder, underline=0)
+        file_menu.add_command(label="Refresh", command=self.refresh, underline=0)
+        file_menu.add_command(label="Exit", command=self.root.destroy, underline=1)
+        menubar.add_cascade(label="File", menu=file_menu, underline=0)
+
+    def open_files(self):
+        files = filedialog.askopenfilenames(
+            filetypes=(
+                ("all files accepted", ["*.png", ".jpg", ".ico", ".gif", ".jpeg"]),
+                ("png files", "*.png"),
+                ("jpg files", "*.jpg"),
+                ("jpeg files", "*.jpeg"),
+                ("gif files", "*.gif"),
+                ("ico files", "*.ico"),
+            )
+        )
+        self.image_files = []
+        self.image_files.append(*files)
+        self.display_image()
+
     def open_folder(self):
         new_dir = filedialog.askdirectory()
-        
-        if new_dir != "": 
+
+        if new_dir != "":
             self.img_dir = new_dir
             self.find_images(self.img_dir)
-    
+
     def refresh(self):
         self.find_images(self.img_dir)
-    
 
     def find_images(self, dir):
-        
+
         self.image_files = []
         self.current_image_index = 0
-        
+
         for item in listdir(dir):
             if isfile(f"{dir}/{item}"):
-                if(item.lower().endswith(('.png', '.jpg', '.jpeg', '.ico', '.gif'))):
-                    self.image_files.append(item)
-                
-        self.display_image()
+                if item.lower().endswith((".png", ".jpg", ".jpeg", ".ico", ".gif")):
+                    self.image_files.append(f"{dir}/{item}")
```

Comment:
I like being able to see the full path, but a slight problem I find with this `self.image_files.append(f"{dir}/{item}")` line is that if you have quite a long file path to an image, then the size of the window changes and then the left and right buttons change position. 

The window change is a pre-existing issue really rather than anything you've done, but I think it's more likely to be an issue with full file paths since they would tend to be longer. Can you change it back to `self.image_files.append(item)` for now please, and I'll create a separate ticket for that issue.

Reply:
This would also mean changing back line 115.